.github/workflows/build-distributions.yml

```
@@ -0,0 +1,48 @@
+name: Build installable distrabutions
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-distribution:
+    strategy:
+      matrix:
+        os: [ macos-latest ] #windows-latest, ubuntu-latest,
+        runner: [
+          {
+            name: arelleCmdLine,
+            executor: arelleCmdLine.py
+          },
+          {
+            name: arelleGUI,
+            executor: arelleGUI.pyw
+          }
+        ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Python 3
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.9
+      - run: pip install -r requirements.txt pyinstaller
+      - uses: actions/checkout@v3
+        with:
+          repository: Arelle/EdgarRenderer
+          path: arelle/plugin/EdgarRenderer
+      - uses: actions/checkout@v3
+        with:
+          repository: xbrlus/xule
+          path: tmp
+      - run: rm -rf tmp/.git && mv tmp/plugin/validate/* arelle/plugin/validate && rm -rf tmp/validate && mv tmp/plugin/* arelle/plugin/* && rm -rf tmp
```

Some of your paths are wrong and there is an unnecessary rm. Also this is easier to read if you use run with a pipe
```suggestion
      - run: |
          mv tmp/plugin/validate/* arelle/plugin/validate/
          rm -rf tmp/plugin/validate
          mv tmp/plugin/* arelle/plugin/
          rm -rf tmp
```